addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2512,7 +2512,7 @@ const Many2oneUserValueWidget = SelectUserValueWidget.extend({
      */
     getValue(methodName) {
         if (methodName === this.options.createMethod && this.createInput) {
-            return this.createInput._value;
+            return this._isAboutCreate ? this.createInput._value : "";
```

Comment:
Lacks inline comments too. The commit message seems clear but the code is not straightforward: here I don't understand: "if I am asking for the value  of the create method and there is a create input... return the create input value" is what there was before and seemed very logical. Now, not sure... why can't I get the create input value at any time I am asking for it?

Reply:
@qsm-odoo I replaced this approach by instead restoring the value when the create button is used (to counter the fact that the value is neutralised by the non-create buttons)